### PR TITLE
Port of Slider Control

### PR DESCRIPTION
**Pull request by eneufeld**
The slider control was contributed with PR #715 . This is a port to the
new repo structure and API using fields

### CODE REVIEW

File: packages/core/test/testers.test.ts [hunk 1]
```diff
@@ -1,213 +1,233 @@
...
+    }
+  };
+  const uischema: ControlElement = {
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/foo'
+    }
+  };
+  t.true(and(
+    schemaTypeIs('string'),
```
Comment (edgarmueller) [inline packages/core/test/testers.test.ts]: Offtopic, but I think we should rename `schemaTypeIs` and `schemaMatches` as we actually mean the schema referenced by the control. Do you agree? If so, we should rename these in a follow-up PR.

File: packages/default/test/renderers/slider.field.test.tsx [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+    <Provider store={store}>
+      <SliderField schema={t.context.schema} uischema={t.context.uischema}/>
+    </Provider>
+  );
+  const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
+  store.dispatch(update('foo', () => null));
+  t.is(input.value, '');
+});
+// FIXME this moves the slider and changes the value
+test('update with wrong ref', t => {
```
Comment (edgarmueller) [inline packages/default/test/renderers/slider.field.test.tsx]: Is the comment accurate? Should this fail?

File: packages/core/test/testers.test.ts (hunk 1 above)
Comment (eneufeld) [reply packages/core/test/testers.test.ts]: Yes I agree, will you open an Issue for this?

File: packages/default/test/renderers/slider.field.test.tsx (hunk 2 above)
Comment (eneufeld) [reply packages/default/test/renderers/slider.field.test.tsx]: I placed the FIXME at the wrong place, will fix it with the rebase

File: packages/core/test/testers.test.ts (hunk 1 above)
Comment (edgarmueller) [reply packages/core/test/testers.test.ts]: Yes, done, see #734

**Pull request merged**